yente/search/mapping.py

```diff
@@ -91,12 +93,29 @@ def make_entity_mapping(schemata: Optional[Iterable[Schema]] = None) -> Dict[str
             # to facet on them.
             if prop.type.group is not None and not excluded:
                 copy_to.append(prop.type.group)
-            prop_mapping[name] = make_type_field(
-                prop.type,
-                copy_to=copy_to,
-                existing=prop_mapping.get(name, None),
+            prop_name_to_fields[name].append(
+                make_type_field(prop.type, copy_to=copy_to)
             )
 
+    prop_mapping: Dict[str, MappingProperty] = {}
+    for prop_name, fields in prop_name_to_fields.items():
+        merged_copy_to = list(
+            set(itertools.chain.from_iterable([f["copy_to"] for f in fields]))
```

Comment:
omg they could have just called this flatten...
ok ok it's just one level and doesn't return a list but

Reply:
Yeah it's dumb.